inst/templates/ad_adlb.R

```diff
@@ -0,0 +1,328 @@
+# Name: ADLB
+#
+# Label: Lab Analysis Dataset
+#
+# Input: adsl, lb
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(dplyr)
+library(lubridate)
+library(stringr)
+
+# ---- Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("lb")
+data("adsl")
+
+lb <- convert_blanks_to_na(lb)
+
+# The CDISC Pilot Data contains no SUPPVS data
+# If you have a SUPPLB then uncomment function below
+
+# lb <- derive_vars_suppqual(lb, supplb) %>%
```

Comment:
I think we should see if we can get admiral.test to include a supplb and showcase the new metatools::derive_vars_suppqual function
https://github.com/pharmaverse/admiral.test/issues/24

Reply:
Looks like this is not in scope of admiral anymore so we can drop this topic

Reply:
I'm resolving the conversation. Thank you @bms63 !